tests/regress/x86_vex.c

```diff
@@ -47,12 +47,37 @@ static void test_vmovdqu(void)
     OK(uc_close(uc));
 }
 
+/* https://github.com/unicorn-engine/unicorn/issues/1656 */
+static void test_vex_l(void)
```

Comment:
Thanks for your contribution. The test case under `tests/regress` is not built & run with CI and should have been moved to `tests/unit`. Could you move this test to `tests/unit/test_x86.c`?

Reply:
I enabled “Allow edits and access to secrets by maintainers“, feel free to edit as you see fit!